src/jdk.accessibility/windows/native/include/bridge/AccessBridgeCalls.h

```diff
@@ -31,13 +31,13 @@
  * AccessBridgeCalls.c is available for download from the OpenJDK repository using
  * the following link:
  *
- * http://hg.openjdk.java.net/jdk9/jdk9/jdk/raw-file/tip/src/jdk.accessibility/windows/native/bridge/AccessBridgeCalls.c
+ * https://git.openjdk.org/jdk/blob/master/src/jdk.accessibility/windows/native/bridge/AccessBridgeCalls.c
  *
  * Also note that the API is used in the jaccessinspector and jaccesswalker tools.
```

Comment:
The problem with this is, is that anyone who gets JDK 20 (or 21 the LTS) will be forever more then pointed at the OpenJDK "tip" and if we made an incompatible ABI change, that would be a problem.
At this point I'd prefer that this be updated to point to JDK 17, as in
https://github.com/openjdk/jdk17/blob/master/src/jdk.accessibility/windows/native/jaccesswalker/jaccesswalker.cpp
So it is a defined, known, compatible version.

Reply:
Updated to refer to JDK 17 specifically.